### PR TITLE
Add documentation for using MinIO with Veeam

### PR DESCRIPTION
**Pull request by eco-minio**
Not using template on purpose, adds some integration documentation for Veeam

### CODE REVIEW

**Comment by minio-trusted** [comment]
```
Error running mint automation
+ mkdir -p 9355-b52b7bf/gopath/src/github.com/minio
+ git -C 9355-b52b7bf/gopath/src/github.com/minio clone --quiet https://github.com/minio/minio.git
+ git -C 9355-b52b7bf/gopath/src/github.com/minio/minio remote add minio https://github.com/minio/minio.git
+ git -C 9355-b52b7bf/gopath/src/github.com/minio/minio fetch --quiet minio pull/9355/head:pr9355
+ git -C 9355-b52b7bf/gopath/src/github.com/minio/minio checkout --quiet pr9355
+ GO111MODULE=on
+ GOPATH=/home/mint-auto/mint-auto/9355-b52b7bf/gopath
+ make -C 9355-b52b7bf/gopath/src/github.com/minio/minio --quiet
go: cannot find main module, but found vendor/vendor.json in /home/mint-auto/mint-auto/9355-b52b7bf/gopath/src/github.com/minio/minio
	to create a module there, run:
	go mod init
make: *** [build] Error 1

```

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-gateway-azure.sh | :x: [more...](#21f70d5d-1e35-4a0c-9b91-7b4ffa2350a1) |
<a name="21f70d5d-1e35-4a0c-9b91-7b4ffa2350a1"></a>
### 9355-8c7218c/mint-gateway-azure.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-c3.minio.io:30474
ACCESS_KEY:           minioazure
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp fb044ff644d4:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 8 seconds
(2/15) Running aws-sdk-java tests ... done in 1 seconds
(3/15) Running aws-sdk-php tests ... done in 1 minutes and 52 seconds
(4/15) Running aws-sdk-ruby tests ... done in 15 seconds
(5/15) Running awscli tests ... done in 3 minutes and 12 seconds
(6/15) Running healthcheck tests ... done in 0 seconds
(7/15) Running mc tests ... done in 6 minutes and 14 seconds
(8/15) Running minio-dotnet tests ... done in 1 minutes and 29 seconds
(9/15) Running minio-go tests ... done in 19 minutes and 1 seconds
(10/15) Running minio-java tests ... done in 26 minutes and 28 seconds
(11/15) Running minio-js tests ... FAILED in 4 minutes and 34 seconds
{
  "name": "minio-js",
  "function": "listObjectsV2WithMetadata(bucketName, prefix, recursive, startAfter)",
  "args": "bucketName:minio-js-test-feab14c9-6697-43b4-9dfa-1bc00036de5e, recursive:true",
  "duration": 212,
  "status": "FAIL",
  "error": "TypeError: Cannot read property '0' of undefined at /mint/run/core/minio-js/node_modules/minio/dist/main/xml-parsers.js:414:42 at Array.forEach (<anonymous>) at parseListObjectsV2WithMetadata (node_modules/minio/dist/main/xml-parsers.js:409:21) at DestroyableTransform._flush (node_modules/minio/dist/main/transformers.js:88:19) at DestroyableTransform.prefinish (node_modules/readable-stream/lib/_stream_transform.js:129:10) at prefinish (node_modules/readable-stream/lib/_stream_writable.js:611:14) at finishMaybe (node_modules/readable-stream/lib/_stream_writable.js:620:5) at endWritable (node_modules/readable-stream/lib/_stream_writable.js:643:3) at DestroyableTransform.Writable.end (node_modules/readable-stream/lib/_stream_writable.js:571:22) at IncomingMessage.onend (_stream_readable.js:673:10) at endReadableNT (_stream_readable.js:1201:12) at processTicksAndRejections (internal/process/task_queues.js:84:21)"
}

Executed 10 out of 15 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Review by harshavardhana — APPROVED** [review]

File: docs/integrations/veeam/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+5. Follow the wizard.  Under the "Object Storage Backup Repository" section, choose the MinIO object storage you created above
+
+![Adding Object Storage to VBO Backup Repository](screenshots/6_add_sobr_with_object_store.png)
+
+6. When you create your backup job, choose the backup repository you created above.  
+
+
+## Test the setup
+The next time the a backup job runs, you can use the  `mc admin trace myminio` command and verify traffic is flowing to the MinIO nodes. For Veeam Backup and Replication you will need to wait for the backup to complete to the performance tier before it migrates data to the capacity tier (i.e., MinIO).
```
Comment (kannappanr) [inline docs/integrations/veeam/README.md]:
```suggestion
The next time the backup job runs, you can use the  `mc admin trace myminio` command and verify traffic is flowing to the MinIO nodes. For Veeam Backup and Replication you will need to wait for the backup to complete to the performance tier before it migrates data to the capacity tier (i.e., MinIO).
```

**Review by kannappanr — APPROVED** [review]

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-gateway-azure.sh | :x: [more...](#075beffd-0d6d-4cc3-a6bb-c46b47242cda) |
<a name="075beffd-0d6d-4cc3-a6bb-c46b47242cda"></a>
### 9355-ec9969b/mint-gateway-azure.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-c3.minio.io:31270
ACCESS_KEY:           minioazure
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp e189c0d89511:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 8 seconds
(2/15) Running aws-sdk-java tests ... done in 2 seconds
(3/15) Running aws-sdk-php tests ... done in 2 minutes and 1 seconds
(4/15) Running aws-sdk-ruby tests ... done in 16 seconds
(5/15) Running awscli tests ... done in 3 minutes and 20 seconds
(6/15) Running healthcheck tests ... done in 0 seconds
(7/15) Running mc tests ... done in 6 minutes and 32 seconds
(8/15) Running minio-dotnet tests ... done in 1 minutes and 39 seconds
(9/15) Running minio-go tests ... done in 19 minutes and 48 seconds
(10/15) Running minio-java tests ... done in 26 minutes and 18 seconds
(11/15) Running minio-js tests ... FAILED in 4 minutes and 33 seconds
{
  "name": "minio-js",
  "function": "listObjectsV2WithMetadata(bucketName, prefix, recursive, startAfter)",
  "args": "bucketName:minio-js-test-4d81227e-0f29-4614-be90-9a76ce9856f0, recursive:true",
  "duration": 167,
  "status": "FAIL",
  "error": "TypeError: Cannot read property '0' of undefined at /mint/run/core/minio-js/node_modules/minio/dist/main/xml-parsers.js:414:42 at Array.forEach (<anonymous>) at parseListObjectsV2WithMetadata (node_modules/minio/dist/main/xml-parsers.js:409:21) at DestroyableTransform._flush (node_modules/minio/dist/main/transformers.js:88:19) at DestroyableTransform.prefinish (node_modules/readable-stream/lib/_stream_transform.js:129:10) at prefinish (node_modules/readable-stream/lib/_stream_writable.js:611:14) at finishMaybe (node_modules/readable-stream/lib/_stream_writable.js:620:5) at endWritable (node_modules/readable-stream/lib/_stream_writable.js:643:3) at DestroyableTransform.Writable.end (node_modules/readable-stream/lib/_stream_writable.js:571:22) at IncomingMessage.onend (_stream_readable.js:673:10) at endReadableNT (_stream_readable.js:1201:12) at processTicksAndRejections (internal/process/task_queues.js:84:21)"
}

Executed 10 out of 15 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Pull request merged**